src/tooltip/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+class Tooltip extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      showTooltip: false
+    };
+    this.handleRenderTooltip = this.handleRenderTooltip.bind(this);
+    this.handleCalculateTooltipPosition = this.handleCalculateTooltipPosition.bind(this);
+  }
+
+  shouldComponentUpdate (nextProps, nextState) {
+    const { showTooltip } = this.state;
+    if (nextState.showTooltip === showTooltip) {
+      return false;
+    }
+    return true;
+  }
+
+  handleRenderTooltip () {
+    const { showTooltip } = this.state;
+    this.setState({ showTooltip: !showTooltip });
+  }
+
+  handleCalculateTooltipPosition (event) {
+    const tooltipClassName = 'tooltip-wrapper';
+    let tooltip = document.getElementsByClassName(tooltipClassName);
+    const xPositionOffset = 25;
+    const yPositionOffset = 65;
```

Comment:
try to replace this with transforms so it works with the dynamic sizes of tooltips (put the transforms in the less file), should be `transform: translate(50%,-100%)` i think